extension/services/deezer/player.js

```diff
@@ -124,6 +140,39 @@ this.player = (function() {
         iconVolumeParent.click();
       }
     }
+
+    async playSection({ query, thenPlay, section }) {
+      try {
+        let SECTION;
```

Comment:
This shouldn't be an upper class variable, that should only be used for constants or global config variables, not a mutable local variable. Maybe `foundSection` would be a good name?